pkg/distributor/billing.go

```diff
@@ -22,7 +22,7 @@ func (d *Distributor) emitBillingRecord(ctx context.Context, buf []byte, samples
 
 	now := time.Now().UTC()
 	hasher := sha256.New()
-	hasher.Write(buf)
+	_, _ = hasher.Write(buf)
```

Comment:
Could add to the project `.errcheck-exclude`

Reply:
@bboreham Not having any luck getting this method added to .errcheck-exclude. I think this is the problem 

https://github.com/golangci/golangci-lint/issues/656

Are you ok with leaving it or would you rather see a comment and I can call out that issue.

Reply:
I think I had a lot of trouble stemming from the way it doesn't re-read the config file unless it thinks something else has changed.  So you need to touch the file or `make clean` or something like that.  I can't remember exactly.

Reply:
Ah ok... I'll give that a go and see what happens. Thanks

Reply:
@zendern May you do it, please?